pages/01.HDM Documentation/06.vcd/10.Network Planning/05.Cloud Network Config/default.md

```diff
@@ -56,20 +61,31 @@ This network will be created for HDM Appliance communications and for migrating
 
 ![alt_text](images/image9.png "image_tooltip")
 
-#### <a name="configure-vpn-tunnel-edge"> Step 3: Configure VPN tunnel with Edge Gateway </a>
+### <a name="configure-vpn-tunnel-edge"> Step 3: Configure VPN tunnel with Edge Gateway </a>
 
 Create an IPSec tunnel between the organization's edge gateway and the on-premises datacenter. To complete the VPN setting the following information will be required:
 
-*   Local ID - Edge gateway tenant external network IP 
-*   Local Endpoint - Edge gateway tenant external network IP
+*   Local ID - Edge gateway tenant external network IP (see _Figure 1_)
+*   Local Endpoint - Edge gateway tenant external network IP (see _Figure 1_)
 *   Local Subnets - Routed network subnet where HDM appliances will be deployed ([Create Routed Network](#create-routed-network))
 *   Peer ID - Public IP for the on-premises WAN network
 *   Peer Endpoint - Public IP for the on-premises WAN network
 *   Peer Subnets - Subnet of the on-premises WAN network where HDM appliances will be deployed.
 
-### Edge gateway VPN config 
-
+_Figure 1: Finding tenant external network IP_
+![Find external IP](images/external-ip.png?classes=content-img "Find external IP")
+#### Edge gateway VPN config 
+To get to the edge VPN configuration page.
+1. Select the services on the edge gateway. 
+![Select services](images/select-service.png?classes=content-img "select services")
```

Comment:
public IP is not masked here too.

Reply:
Again, not our public IP I have modified it.